src/bootstrap.js

```diff
@@ -4,9 +4,35 @@
 
 /* eslint-disable no-unused-vars */
 
+const { utils: Cu } = Components;
+
+Cu.import("resource://gre/modules/Services.jsm");
+
 function startup({webExtension}, reason) {
-  webExtension.startup().then(() => {
+  Services.telemetry.registerEvents("lockbox", {
```

Comment:
I don't know if an external JSON file is worthwhile.  Each event will involve corresponding code changes to actually record the event.